src/resources/schema2md/README.md

```diff
@@ -0,0 +1,61 @@
+# README
+
+## Top-level Schemas
+
+* [Blockchain address](./blockchain-address.md "An address on a blockchain") – `blockchain-address.json`
+
+* [Collection](./collection.md "A collection of projects") – `collection.json`
+
+* [Named Entity](./named.md "A generic named object") – `named.json`
+
+* [Project](./project.md "A project is a collection of artifacts") – `project.json`
+
+* [Social Profile](./social-profile.md "All social profile") – `social-profile.json`
+
+* [URL](./url.md "A generic URL") – `url.json`
+
+## Other Schemas
+
+### Objects
+
+
+
+### Arrays
+
+* [Untitled array in Blockchain address](./blockchain-address-properties-tags.md) – `blockchain-address.json#/properties/tags`
```

Comment:
Hrm `Untitled array` is maybe not the most useful. I'm not sure that having `Arrays` as a top level item is generally useful either. I think we should instead only document the objects inside the array if they're object types and not any built in type.

Reply:
Is this something we could _change_ in jsonschema2md?